apps/api/.env.example

```diff
@@ -23,4 +23,11 @@ TWITTER_REDIRECT_URI=
 TWITTER_CLIENT_ID=
 TWITTER_CLIENT_SECRET=
 
-BLOCKCHAIN_SEPOLIA_RPC_URL=
+# The network name must be the same as what appears in the `blockchainCredentialSupportedNetworks` list 
```

Comment:
it might make sense to add an example 

# e.g., <example-here>

Reply:
Thanks! 

You mean, after this phrase: `instead of a space to separate words.`, add `e.g. Polygon Mumbai would be POLYGON_MUMBAI`?

Reply:
exactly!

Reply:
Perfect, thanks. I will update it 👍